apps/condo/domains/ticket/utils/helpers.ts

```diff
@@ -5,6 +5,7 @@ import { LOCALES } from '@condo/domains/common/constants/locale'
 import moment from 'moment'
 import { ParsedUrlQuery } from 'querystring'
 import { Ticket, TicketStatus, TicketStatusWhereInput, TicketWhereInput } from '../../../schema'
+import { ITicketAnalyticsPageFilters } from '../../../pages/analytics'
```

Comment:
strange that you include something to domains from pages 
maybe, you can move these declarations to the domain?

Reply:
@dkoviazin Agree with you, but this types mostly used at client side, so I think it should be specified here

Reply:
Filter to query should declare ITicketAnalyticsPageFilters interface near function declaration, page should use this interface.

Reply:
Move it here, please

Reply:
@dkoviazin, @Dimitreee done